gpustack/schemas/models.py

```diff
@@ -412,6 +443,54 @@ class ModelInstanceBase(SQLModel, ModelSource):
 
     cluster_id: Optional[int] = Field(default=None, foreign_key="clusters.id")
 
+    @lru_cache
```

Comment:
Need to check this part:
Do we need to define `__eq__`, `__hash__` for the class: https://docs.python.org/3/faq/programming.html#faq-cache-method-calls
Any possible issues using fuctools with instance methods: https://stackoverflow.com/questions/33672412/python-functools-lru-cache-with-instance-methods-release-object

Reply:
I found that instance methods use `lru_cache` in many frameworks: transformer, rich. But I get your point, remoe `lru_cache` now, it seems that `get_deployment_metadata` only call once.